packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -289,55 +295,53 @@ export default function ProjectNav({
             active={open}
             disabled={!current}
             data-testid="project-nav-project-menu"
+            onClick={onElementClick(ProjectNavProjectDropdown)}
           >
             <Icon glyph="Ellipsis" className={menuIconStyle} />
           </IconButton>
         }
       >
         <MenuItem
           href={projectNav.settings}
-          onClick={onMenuClick}
           data-testid="project-nav-settings"
+          onClick={onElementClick(ProjectNavProjectSettings, () =>
```

Comment:
FYI could have also done: 
```suggestion
          onClick={onElementClick(ProjectNavProjectSettings, onMenuClick)}
```

Reply:
I can revert if you prefer, lmk

Reply:
Nah, up to you, just letting you know alternative ways to solve the problem.